test/unit/webdriver/nativekey_test.py

```diff
@@ -0,0 +1,30 @@
+from appium.webdriver.extensions.android.nativekey import AndroidKey
+
+
+class TestAndroidKey:
+
+    def test_has_some_codes(self):
+        assert AndroidKey.ENTER == 66
+        assert AndroidKey.BACK == 4
+        assert AndroidKey.CAMERA == 27
+        assert AndroidKey.SPACE == 62
+
+    def test_is_gamepad_key(self):
+        assert AndroidKey.is_gamepad_button(195)
```

Comment:
I prefer `Android.XXX` instead of 195. It's more clear as test cases.

Same comment to below.